public/404.html

```diff
@@ -58,10 +62,17 @@
   <!-- This file lives in public/404.html -->
   <div class="dialog">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
+      <h2>Publify</h2>
+      <h1>404 Not Found</h1>
       <p>You may have mistyped the address or the page may have moved.</p>
     </div>
-    <p>If you are the application owner check the logs for more information.</p>
+    <p>If you are the application owner check the logs for more information.
+    <br>
+    <br>
+    <a href="http://localhost:3000/">Home</a> |
```

Comment:
Tricky. You could simply change the links to just ba a path: `/`, `/notes`, and `/archives`, but in particular notes may not be relevant, and this will break if the blog is not located at the root of the web server.

Removing the links is probably the best option.